packages/js/product-editor/src/utils/get-related-products/index.ts

```diff
@@ -4,20 +4,65 @@
 import { select, resolveSelect } from '@wordpress/data';
 import type { Product } from '@woocommerce/data';
 
-export default async function getRelatedProducts( productId: number ) {
+type getRelatedProductsOptions = {
+	// If true, return random products if no related products are found.
+	fallbackToRandomProducts?: boolean;
+};
+
+const POSTS_NUMBER_TO_RANDOMIZE = 30;
+
+/**
+ * Return related products for a given product ID.
+ * If fallbackToRandomProducts is true,
+ * return random products if no related products are found.
+ *
+ * @param {number}  productId - Product ID.
+ * @param {boolean} options   - Options.
```

Comment:
I think that `options` is an object, not a boolean.

Reply:
Thanks. Actually we can use the `getRelatedProductsOptions` type

Reply:
92b0f07cbfc42eb96e8578535603ef3a6a5a17d2